.travis.yml

```diff
@@ -3,8 +3,18 @@ os:
   - linux
   - osx
 node_js:
+  - "8.11.3"
```

Comment:
Since our docs say `8.11.3` is our minimum supported node.js version, and since there are newer versions of `8.11.x`, then we should continue running CI against it. We can *probably* trust that the Node maintainers respects semver, but this will actually become more useful later on. `8.11.3` ships with an older npm v5, and as we evolve our test dapps to test more realistically against a build of embark installed with npm, it will be good to know if using an older version of npm causes problems when installing and running embark.

Reply:
We should probably change the doc to specify `8.11.x`

Reply:
In my comment I wrote:
> ... newer versions of `8.11.x` ...
But I actually meant `8.x`, since `8.12.0` is the latest in the `8.x` LTS release line.

I think it is an interesting discussion to have if we want to decide and document that our minimum supported version is whatever the latest is in some LTS release line vs. a fixed version.

Reply:
Agree this is something we can discussed, also that would make our list of version of node we support very easy to follow